docs/integrate/quick-start/electron.en.md

```diff
@@ -26,30 +26,30 @@ Run the following commands in sequence:
 ```bash
 $ git clone git@github.com:opensumi/ide-electron.git
 $ cd ide-electron
-$ npm install
-$ npm run build
-$ npm run rebuild-native -- --force-rebuild=true
-$ npm run download-extension # install built-in extensions (optional)  
-$ npm run start
+$ yarn
+$ yarn run build
```

Comment:
some scripts like `yarn run build` can be shortly written as `yarn build`,  maybe you can also create a PR for `ide-electron`.

Reply:
done.